src/Feedback.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Gorse;
+
+class Feedback implements \JsonSerializable
+{
+    public function __construct(
+        public string $feedback_type,
+        public string $user_id,
```

Comment:
Also I think camelCase is much common now than snake_case

Reply:
Regarding this, the current coding style is being maintained as much as possible to prevent compatibility issues. The implementation of camel case naming will be considered in future major versions.